code/modules/client/preference_setup/loadout/loadout_shoes.dm

```diff
@@ -5,6 +5,11 @@
 	slot = slot_shoes
 	sort_category = "Shoes and Footwear"
 
+/datum/gear/shoes/whiteboots
+	display_name = "white jackboots"
+	path = /obj/item/clothing/shoes/boots/jackboots/white
+	slot = slot_shoes
```

Comment:
Could you elaborate on that? I don't quite see what's out of place or unnecessary.